### PR TITLE
mgr/nfs: add more debug log msg

### PR DESCRIPTION
**Pull request by varshar16**
Fixes: https://tracker.ceph.com/issues/52274

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by varshar16** [comment]
jenkins test make check

File: src/pybind/mgr/nfs/export.py [hunk 1]
```diff
@@ -537,6 +540,7 @@ def create_export_from_dict(self,
         ex_dict["cluster_id"] = cluster_id
         export = Export.from_dict(ex_id, ex_dict)
         export.validate(self.mgr)
+        log.debug(f"Succefully created {fsal_type} export-{ex_id} from dict for cluster {cluster_id}")
```
Comment (batrick) [inline src/pybind/mgr/nfs/export.py:543]:
```suggestion
        log.debug(f"Successfully created {fsal_type} export-{ex_id} from dict for cluster {cluster_id}")
```

**Review by batrick — CHANGES_REQUESTED** [review]

File: src/pybind/mgr/nfs/export.py (hunk 1 above)
Comment (varshar16) [reply src/pybind/mgr/nfs/export.py]: Done

**Review by batrick — APPROVED** [review]

File: src/pybind/mgr/nfs/cluster.py [hunk 2]
```diff
@@ -41,6 +41,7 @@ def create_ganesha_pool(mgr: 'MgrModule') -> None:
         mgr.check_mon_command({'prefix': 'osd pool application enable',
                                'pool': POOL_NAME,
                                'app': 'nfs'})
+        log.debug(f"Successfully created nfs-ganesha pool {POOL_NAME}")
```
Comment (alfonsomthd) [inline src/pybind/mgr/nfs/cluster.py:44]: Just an observation: [it's recommended to use Lazy Logging](https://medium.com/swlh/why-it-matters-how-you-log-in-python-1a1085851205) (in Dashboard module the linter complains about this).

**Pull request closed without merging**